src/app/(frontend)/embeds/providers/page.tsx

```diff
@@ -0,0 +1,117 @@
+import { ProviderPreview } from '@/components/ProviderPreview'
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from '@/components/ui/accordion'
+import { getStateLabel } from '@/fields/location/states'
+import type { Provider } from '@/payload-types'
+import config from '@/payload.config'
+import { createLoader, parseAsString, SearchParams } from 'nuqs/server'
+import { getPayload } from 'payload'
+
+export const dynamic = 'force-static'
+
+const providersSearchParams = {
+  backgroundColor: parseAsString,
+  title: parseAsString,
+}
+
+const loadSearchParams = createLoader(providersSearchParams)
+
+type ProvidersByState = { [state: string]: Provider[] }
+
+export default async function ProvidersEmbedPage({
+  searchParams,
+}: {
+  searchParams: Promise<SearchParams>
+}) {
+  const { backgroundColor, title } = await loadSearchParams(searchParams)
+
+  const payload = await getPayload({ config })
+
+  const result = await payload.find({
+    collection: 'providers',
+    where: {
+      _status: {
+        equals: 'published',
+      },
+    },
+    limit: 1000,
+    depth: 0,
+  })
+
+  const providers = result.docs as Provider[]
+
+  // Organize providers by state based on statesServiced
+  // providers can be in multiple states
+  const providersByState: ProvidersByState = {}
+
+  providers.forEach((provider) => {
+    if (provider.statesServiced && provider.statesServiced.length > 0) {
+      provider.statesServiced.forEach((state) => {
+        if (!providersByState[state]) {
+          providersByState[state] = []
+        }
+        providersByState[state].push(provider)
+      })
+    }
+  })
+
+  const states = Object.keys(providersByState).sort()
+
+  // Split states into two columns for vertical flow
+  const midpoint = Math.ceil(states.length / 2)
+  const leftColumnStates = states.slice(0, midpoint)
+  const rightColumnStates = states.slice(midpoint)
```

Comment:
Reminder to remove this based on https://github.com/NWACus/web/pull/675/files#r2530430838